### ISSUE TITLE
Font size does not match computer settings

### ISSUE THREAD

**Issue by amias-scribe** [ffes/nppsnippets#57]
Hi! I use an increased text size on my computer. NPP itself respects this setting in its UI and has a zoom setting for the editor, so I am able to adjust it comfortably. However, Snippets does not resize the text under any circumstances I can find. It would be great if Snippets followed either the Windows font size or the NPP zoom!

**Comment by waltwooton** [comment]
Yes. This is my only complaint with Snippets. The text is way too small. It's great otherwise.
